Marlin/src/inc/SanityCheck.h

```diff
@@ -1707,7 +1707,7 @@ static_assert(hbm[Z_AXIS] >= 0, "HOMING_BUMP_MM.Z must be greater than or equal
 #endif
 
 #if HAS_MULTI_HOTEND
-  #if HEATER_1_USES_MAX6675 && !PIN_EXISTS(MAX6675_SS2)
+  #if HEATER_1_USES_MAX6675 && !PIN_EXISTS(MAX6675_SS2) && !(PIN_EXISTS(MAX31855_CS2) || PIN_EXISTS(MAX31865_CS2) || PIN_EXISTS(MAX6675_CS2))
```

Comment:
CS/SS again

Reply:
Since @thinkyhead has already done work on this area, I defer to his expertise.

Reply:
I fixed this so that MAX????_SS_PIN is now put into MAX????_CS_PIN in conditionals_post.h file. When SanityCheck gets executed the code will use the MAX????_CS_PIN variables.